src/component/calculator.css

```diff
@@ -2,19 +2,29 @@ body {
   background-color: brown;
 }
 
+.overall {
+  display: flex;
+  justify-content: space-around;
+}
+
```

Comment:
- Please, kindly adjust your styling as a user is finding it difficult to see the numbers they've inputted in your calculator. The screen content of your calculator input is not showing complete. please, kindly fix this issue.

![calculator](https://user-images.githubusercontent.com/59397159/185202722-7eb81b5a-ee4e-44ef-bd2c-60eb6d9507c9.png)

Reply:
* Styling fix.
* Numbers can be seen better now.